### ISSUE TITLE
Tensor name "resnet50_v1d/C1/conv0/BatchNorm/beta" not found in checkpoint files resnet50_v1d.ckpt

### ISSUE THREAD

**Issue by Deepak3994** [DetectionTeamUCAS/NAS_FPN_Tensorflow#6]
Hi,

I was trying to train NAS-FPN model for my classes. But could not able to load "Tensor name "resnet50_v1d/C1/conv0/BatchNorm/beta" not found in checkpoint files resnet50_v1d.ckpt".

Note:
NUM_CLASSES = 1
Image_size = (500,500)
Back_bone i am using = resnet50_v1d.ckpt
For detection i am using pre-trained model = FPN_Res50_COCO_20190429_v3

Kindly help.
